mne/preprocessing/maxwell.py

```diff
@@ -0,0 +1,255 @@
+# Authors: Mark Wronkiewicz <wronk.mark@gmail.com>
+#          Jussi Nurminen
+#
+# License: BSD (3-clause)
+
+#TODO: write in equation numbers from Samu's paper
+
+import numpy as np
+from scipy.special import sph_harm, lpmv
+from scipy.misc import factorial
+from ..forward._lead_dots import (_get_legen_table, _get_legen_lut_fast,
+                                  _get_legen_lut_accurate)
+from ..fixes import partial
+from ..transforms import _cartesian_to_sphere as cart_to_sph
+from ..transforms import _sphere_to_cartesian as sph_to_cart
+from scipy.misc import factorial as fact
+
+
+def maxwell_filter(Raw, origin, L_in, L_out):
+    """
+    Filter data using SSS basis functions.
+
+    Parameters
+    ----------
+    Raw : mne.Raw instance
+        Data to be filtered.
+    origin : array of shape [3,]
+        Origin of head.
+    L_in : int
+        Order of in-component spherical expansion
+    L_out : int
+        Order of out-component spherical expansion
+
+    Returns
+    -------
+    sss_Raw : Filtered data
```

Comment:
eventually, yes, but there might actually be different/better algorithms that we could use in that case -- so it'll be a different PR at the very least :)